pydantic_mongo/abstract_repository.py

```diff
@@ -69,8 +69,11 @@ def to_document(model: T) -> dict:
         """
         data = model.model_dump()
         data.pop("id")
+        data.pop("v")
```

Comment:
This isn't related to kwargs feature, can you open it as a separate PR? My intention is to publish simple changes and break compat in different versions.